### DISCUSSION TITLE
Data Model

### DISCUSSION THREAD

**Discussion by Shair17** [prisma/prisma#9146] [Q&A] [answered]
Is there a site where they help to write models of common things like comments and answers, ranking, levels and so on but in prismajs?

**Comment by ryands17** [comment, marked as answer]
@Shair17 👋 

There's no site like this but you can ask your question here regarding data modelling. You can also have a look at the following links to understand how data modelling in Prisma works:
https://www.prisma.io/docs/concepts/components/prisma-schema/data-model
https://www.prisma.io/docs/concepts/components/prisma-schema/relations

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
